src/node_crypto.cc

```diff
@@ -865,7 +885,8 @@ void SecureContext::SetSessionTimeout(const FunctionCallbackInfo<Value>& args) {
   SecureContext* sc = Unwrap<SecureContext>(args.Holder());
 
   if (args.Length() != 1 || !args[0]->IsInt32()) {
-    return sc->env()->ThrowTypeError("Bad parameter");
+    return sc->env()->ThrowTypeError(
+        "Session Timeout must be a 32-bit integer");
```

Comment:
"Session timeout" and "PFX certificate argument" a few lines below.  I'm going to stop pointing those out from now on.